bfsysroot/bfunwind/include/dwarf4.h

```diff
@@ -357,6 +638,17 @@ class dwarf4
     ///
     static int64_t decode_sleb128(char **addr);
 
+    /// Decode Signed LEB128
```

Comment:
the use of this function is fine, but I would like to find a way to have the two functions use the same code (i.e. the original function should call this one and then add to addr as needed.